.github/workflows/code-review.yml

```
@@ -0,0 +1,71 @@
+name: AI Code Review
+
+on:
+  push:
```

actionlint (1.7.10)

[error] "types" cannot be specified for "push" Webhook event

(events)